src/main/resources/default/templates/biz/jobs/interactive-job.html.pasta

```diff
@@ -13,84 +17,73 @@
         </li>
     </i:block>
 
-    <w:pageHeader>
-        <div class="row">
-            <div class="col-md-8">
-                @job.getCurrentLabel(context)
+    <t:pageHeader>
+        <i:block name="title">@job.getCurrentLabel(context)</i:block>
+        <t:iconInlineInfo icon="fa fa-info-circle">@job.getCurrentDescription(context)</t:iconInlineInfo>
+        <i:block name="actions">
+            <i:if test="!job.getJobInfos().isEmpty()">
+                <a class="btn btn-outline-link" href="/jobs/infos/@job.getName()">
+                    <span><i class="fa fa-book"></i></span> @i18n("JobsController.showInfos")
+                </a>
+            </i:if>
+        </i:block>
+    </t:pageHeader>
```

Comment:
```suggestion
    <i:block name="page-header">
        <t:pageHeader>
            <i:block name="title">@job.getCurrentLabel(context)</i:block>
            <t:iconInlineInfo icon="fa fa-info-circle">@job.getCurrentDescription(context)</t:iconInlineInfo>
            <i:block name="actions">
                <i:if test="!job.getJobInfos().isEmpty()">
                    <a class="btn btn-outline-link" href="/jobs/infos/@job.getName()">
                        <span><i class="fa fa-book"></i></span> @i18n("JobsController.showInfos")
                    </a>
                </i:if>
            </i:block>
        </t:pageHeader>
    </i:block>
```

Reply:
diff is pretty large, but this is only wrapping the page header in the appropriate block